Dependencies/xr/Source/OpenXR/SceneUnderstanding.cpp

```diff
@@ -5,11 +5,32 @@
 #include "SceneUnderstanding.h"
 #include "XrRegistry.h"
 
+#pragma region DirectX

```

Comment:
Typically in other parts of Babylon Native we've split these sorts of things out into platform-specific headers/implementations that are included as appropriate by CMake. Could that approach be used for this?

Reply:
this seems reasonable. The directx specific region was a weaker attempt at this :)

Reply:
moved to XrPlatform.h, which is windows specific